core/src/main/scala/kafka/server/DynamicConfigManager.scala

```diff
@@ -141,8 +146,12 @@ class DynamicConfigManager(private val zkUtils: ZkUtils,
               " Received: " + json)
       val fullSanitizedEntityName = entityPath.substring(index + 1)
 
+      def loggableConfig(entityConfig: Properties) = entityConfig.asScala.map {
```

Comment:
Is there a reason why we made this a `def` instead of just having a `val` after `entityConfig`?